internal/controller/scheduler/scheduler.go

```diff
@@ -146,15 +146,7 @@ func (w *worker) Handle(ctx context.Context, job *api.AgentScheduledJob) error {
 		return w.failJob(ctx, inputs, fmt.Sprintf("agent-stack-k8s failed to parse the job: %v", err))
 	}
 
-	// Default command container using default image.
-	podSpec := &corev1.PodSpec{
-		Containers: []corev1.Container{
-			{
-				Image:   w.cfg.Image,
-				Command: []string{jobToRun.Command},
-			},
-		},
-	}
```

Comment:
I believe this change is reasonable as we are doubling down agent as a parent process strategy. 

However, I can imagine that it will impact some users who do not want agent to be the parent process. I think as we are recently about to release a number of breaking change, this is a good moment to draw a line.